level-set_advection_reinitialization.cc

```diff
@@ -3471,13 +3492,16 @@ namespace LevelSet
             time_step_advection = param.FINAL_TIME - time;
           }
 
-          time_integrator.perform_time_step(level_set_operator,
-                                            time,
-                                            time_step_advection,
-                                            solution,
-                                            rk_register_1,
-                                            rk_register_2,
-                                            computing_timer);
+          //time_integrator.perform_time_step(level_set_operator,
+          //                                  time,
+          //                                  time_step_advection,
+          //                                  solution,
+          //                                  rk_register_1,
+          //                                  rk_register_2,
+          //                                  computing_timer);
```

Comment:
Could you add a parameter for the `time_integration_scheme` such that the explicit integration is still possible?